_opensearch/query-dsl/full-text.md

```diff
@@ -395,40 +395,7 @@ GET _search
 
 ## Convert text with analyzers
 
-OpenSearch provides the analyzer option to convert your structured text into the format that works best for your searches. You can use the following options with the analyzer field: standard, simple, whitespace, stop, keyword, pattern, fingerprint, and language. Different analyzers have different character filters, tokenizers, and token filters. The stop analyzer, for example, removes stop words (e.g., “an,” “but,” “this”) from the query string.
-
-OpenSearch supports the following language values with the `analyzer` option:
-arabic, armenian, basque, bengali, brazilian, bulgarian, catalan, czech, danish, dutch, english, estonian, finnish, french, galicia, german, greek, hindi, hungarian, indonesian, irish, italian, latvian, lithuanian, norwegian, persian, portuguese, romanian, russian, sorani, spanish, swedish, turkish, and thai.
-
-To use the analyzer when you map an index, specify the value within your query. For example, to map your index with the French language analyzer, specify the `french` value for the analyzer field:
-
-```json
- "analyzer": "french"
- ```
-
-#### Sample Request
-
-The following query maps an index with the language analyzer set to `french`:
-
-```json
-PUT my-index-000001
-
-{
-  "mappings": {
-    "properties": {
-      "text": { 
-        "type": "text",
-        "fields": {
-          "french": { 
-            "type":     "text",
-            "analyzer": "french"
-          }
-        }
-      }
-    }
-  }
-}
-```
+To convert unstructured text, use the text analyzers. See [Text analyzers]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/text-analyzers).
```

Comment:
I suggest adding to convert to what: To convert unstructured text into the format you need, use the text analyzers. Or something like that.

Reply:
Do we want to mention what text analysis is in general somewhere here? You use text analysis to convert unstructured text into structured text so that it is optimized for search.

Reply:
yes. The concepts are explained on the Text analyzers page.

Reply:
i gave more details in the learn more link: To convert unstructured text _into the format that you need, you can use the text analyzers. To learn more about how to convert unstructured text into structured text that is optimized for search,_ see [Text analyzers]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/text-analyzers).